sixtyfps_compiler/lib.rs

```diff
@@ -87,11 +93,23 @@ impl CompilerConfiguration {
             }
         };
 
+        let inline_all_elements = match std::env::var("SIXTYFPS_DISABLE_INLINING") {
+            Ok(var) => {
+                !var.parse::<bool>().unwrap_or_else(|_|{
+                    panic!("SIXTYFPS_DISABLE_INLINING has incorrect value. Must be either unset, 'true' or 'false'")
+                })
+            }
+            // Currently, the interpreter needs the inlining to be on.
+            //Err(_) => output_format == crate::generator::OutputFormat::Interpreter,
+            Err(_) => false,
```

Comment:
```suggestion
            Err(_) => true,
```

I think `true` needs to be the default until we make this an opt-in (SIXTYFPS_FORCE_INLINING=true or so)

Reply:
Yeah, that was wrong. 
I changed to enable it by default now, unless with the interpreter.